### ISSUE TITLE
i can't run  zlogger 

### ISSUE THREAD

**Issue by Darkbeast-glitch** [z00z/ZLogger#4]
when i try running zlogger i get hist

kali@kali:~/ZLogger$ sudo python zlogger.py -i 60 -w -e bbjulius@gmail.com -p jjkjdkhee -o keylogger
wine: cannot find '/root/.wine/drive_c/Python27/Scripts/pyinstaller.exe'

[***] Don't forget to allow less secure applications in your Gmail account.
Use the following link to do so https://myaccount.google.com/lesssecureapps
kali@kali:~/ZLogger$

**Comment by HarshitRuwali** [comment]
Have you run the `install.sh` file?
Try running `bash install.sh`.
Because it installs all the missing packages and you don't have to work on manually to install packages.

**Comment by HarshitRuwali** [comment]
Have you run the `install.sh` file?
Try running `bash install.sh`.
Because it installs all the missing packages and setup the environment.
And it might be possible that this will solve your problem.

**Comment by ColbyJack1134** [comment]
I am getting the same error above even after running the install.sh file

**Comment by HarshitRuwali** [comment]
Try installing wine.
 Ref this [link](https://computingforgeeks.com/how-to-install-wine-on-kali-linux-and-debian-64-bit/).
Installing wine might take quite a time and the process is tedious if you have broken packages.

**Comment by achraf8989** [comment]
please HarshitRuwali if you can fix the errors i will be very gratiful, i m not specialist in programing but i know follow steps ,

**Comment by HarshitRuwali** [comment]
Try to install wine. I have mentioned the link above. That should help. <br>
If not let me know.

**Comment by achraf8989** [comment]
I have already install it ( i have some error concernin the version of linux when we want install pip ) but i think that we should be very carful with the  version of linux ( new version that will allow to use wine, pip ..) that we have, and the good link for sources.list to get update and install ( wine, pip ...) , what i will do , i will change the kali version first then i will retry to install pip and wine , and if i have some issue i will let you know .

Please if i'm wrong in one steps of those try to suggest me your own steps from the installation of Linux (update ...) to the installation of zlooger , thank you.

**Comment by HarshitRuwali** [comment]
I am currently using Kali 2020 version in the virtual box and its working fine. No error, nothing.

**Comment by achraf8989** [comment]
Cool, OK i will use this version There Is no package to install and and folder creaction apart the wine installation.

**Comment by HarshitRuwali** [comment]
No. Rest of the packages which are not installed will be installed while you will run the `install.sh` file. You just to install wine manually.

**Comment by achraf8989** [comment]
Okk i will do that

**Comment by achraf8989** [comment]
hay harshit i done exactelly what you said to me it works thank you very muuuch but there is a probleme when sending to the email , the symbols are not sent , there is only a fiew word sent and after a 3 min the program cant run and show the message 'failed to execute the program' , what can i do in this case.

**Comment by HarshitRuwali** [comment]
Providing the exact logs might be helpful.

**Comment by achraf8989** [comment]
thank you iven i dont uderstand very well, Please if you can give  me a date which you will availabale 
and i have teamviwer already installed in my pc come and work in my machine thank you

**Comment by achraf8989** [comment]
thank you iven i dont uderstand very well, Please if you can give  me a date which you will availabale 
and i have teamviwer already installed in my pc come and work in my machine thank you

**Comment by HarshitRuwali** [comment]
Well I am referring to the error which you are encountering when you are running the program. 
Just provide the screen-shot and I will look into it. Then only I can understand the problem.

**Comment by Nisarg-202** [comment]
when i run Zlogger.exe file it run successfully done.  But it can't give me a response in email ( i have also on the less secure app access option ) . Please solve my problem.

**Comment by Sadtrxsh** [comment]
its a simple fix 
do:

    sudo add-apt-repository ppa:ubuntu-wine/ppa
    sudo apt-get update'
    sudo apt-get install wine1.6 winetricks
    winetricks python27
    wine python.exe Scripts/pip.exe install pyinstaller

should fix your issue

**Comment by Ali83128** [comment]
I'm still facing this issue can't fix it

**Comment by geniusforyou** [comment]
anyone send me step by step commands to be used pls......!
